src/mumble/widgets/TrayIcon.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright 2024 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "TrayIcon.h"
+
+#include "../ClientUser.h"
+#include "../MainWindow.h"
+#include "../Global.h"
```

Comment:
Couldn't do the include "normally" as these files should be compiled with the regular include path? :thinking: 

Having explicit include paths is always a bit fishy :eyes: